change_log/next/Fix issues with experimental date input.yml

```diff
@@ -0,0 +1 @@
+Styling Changes: "Adjusts the width in classic theme and applies greater width when in DLS themes input is large The `StyledDateInput` now receives a size prop. Component: DateInput)"
```

Comment:
```suggestion
Styling Changes: "Adjusts the width in classic theme and applies greater width when in DLS themes input is large The `StyledDateInput` now receives a size prop. (Component: DateInput)"
```

Reply:
This was you suggested change previously